src/server/session.ts

```diff
@@ -1895,14 +1900,27 @@ namespace ts.server {
 
         private getFullNavigateToItems(args: protocol.NavtoRequestArgs): readonly NavigateToItem[] {
             const { currentFileOnly, searchValue, maxResultCount } = args;
+            if (!args.file) {

```

Comment:
>file [](start = 22, length = 4)

Seems like it's now possible to specify a project and no file.  If some caller were to do that, I'd expect it to get back `project.getLanguageService().getNavigateToItems`.

Reply:
Agree, With no file and just project, the result should be only from that project.

Reply:
The original case handles basically this case, but includes hits from all dependencies as well. Any ideas for how to reduce it to just the current project?

(It's probably OK to keep the original behaviour, since it won't at least surprise anyone.)

Reply:
For example, in the test cases, the 'user' project that depends on 'a' and 'b' projects returns [user.ts, a.ts, b.ts] for `program.getSourceFiles()`